src/cmd/manage_nsfs.js

```diff
@@ -642,6 +644,19 @@ async function list_config_files(type, wide, show_secrets, filters = {}) {
     return config_files_list;
 }
 
+/**
+ * list_connections
+ * @returns An array with names of all connection files.
+ */
+async function list_connections() {
+    let conns = await config_fs.list_connections();
+    // it inserts undefined for the entry '.noobaa-config-nsfs' and we wish to remove it
+    // in case the entry was deleted during the list it also inserts undefined
```

Comment:
Is this comment relevant here?

Reply:
Yes? Same functionality as the original list().